fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1614,14 +1614,6 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = true,
     )
 
-    /**
-     * Indicates if the Task Continuity enhancements are enabled.
-     */
-    var enableTaskContinuityEnhancements by booleanPreference(
-        key = appContext.getPreferenceKey(R.string.pref_key_enable_task_continuity),
```

Comment:
❓I believe we can also remove the `pref_key_enable_task_continuity` as well since there should be no usage of this after this.

Reply:
Good point, updated!